source/navitiacommon/navitiacommon/models/equipments_providers.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, id, json=None):
             self.from_json(json)
 
     def from_json(self, json):
-        self.klass = json['class']
+        self.klass = json['klass']
```

Comment:
this will pose some compatibility issue with tyr-configurator

Reply:
This is actually to be compatible with tyr-configurator:
https://github.com/CanalTP/tyr-configurator/pull/18